src/gamecontroller.cs

```diff
@@ -142,11 +166,11 @@ partial interface GCControllerDirectionPad {
 	[Deprecated (PlatformName.TvOS, 10, 0, message: "Use 'GCExtendedGamepad' instead.")]
 	[iOS (7,0)]
 	[Mac (10,9)]
-	[BaseType (typeof (NSObject))]
+	[BaseType (typeof (GCPhysicalInputProfile))]
 	[DisableDefaultCtor] // return nil handle -> only exposed as getter
 	partial interface GCGamepad {
 
-		[NullAllowed]
+		//[NullAllowed] This seems like it's a breaking change...
```

Comment:
Is this a breaking change? 🤔 It doesn't seem like it. Any code that assumes that a null may be returned will still work here - we're restricting the return value to a subset of what it used to be.

Reply:
I don't think changing this has any effect on our generated code (IOW it's fine).

Reply:
AFAIK it is not an issue at all. It is a get and not a set, so we do not care. @whitneyschmidt on a 'set' we do check for the value and throw a NRE depending on the value, but since we only return, we are fine.

Reply:
Nope not a breaking change